tests/tensor/test_blockwise.py

```diff
@@ -0,0 +1,174 @@
+import numpy as np
+import pytest
+
+import aesara
+import aesara.tensor as at
+from aesara.tensor.blockwise import (
+    Blockwise,
+    _calculate_shapes,
+    _parse_input_dimensions,
+    _update_dim_sizes,
+    gufunc_sign_to_str,
+)
+from aesara.tensor.math import Dot
+from aesara.tensor.type import TensorType
+from tests.unittest_tools import check_infer_shape, verify_grad
+
+
+class DotBW(Dot):
+    gufunc_sig = ((("m", "n"), ("n", "p")), (("m", "p"),))
+
+
+dot_bw = DotBW()
```

Comment:
If `Dot` has been given a signature, we can remove this special subclass and instance.